api/path_service_config.go

```diff
@@ -466,7 +466,14 @@ func CreateService(service *Service,
 	if err != nil {
 		glog.V(2).Infof("Failed to retrieve node policy from local db: %v", err)
 	}
-	externalPol, _ := externalpolicy.CreateNodeBuiltInPolicy(false, false, existingPol)
+	isCluster := false
+	exchDevice, err := persistence.FindExchangeDevice(db)
```

Comment:
The variable pDevice already has device object from the local DB. I think you can just use that instead of reading it again.

Reply:
fixed